### DISCUSSION TITLE
The coredns custom configuration in k3s seems to revert to the default configuration after restarting k3s, what should I do to make the custom configuration permanent after k3s restart?

### DISCUSSION THREAD

**Discussion by MagicStarTrace** [k3s-io/k3s#7633] [Q&A]
My system is CentOS 7
k3s version v1.21.5+k3s2 (724ef700)

**Comment by brandond** [comment]
Any of the manifests packaged with k3s will be restored to default when the server restarts. If what you want to do can't be done via a custom server block, or the overrides requested at https://github.com/k3s-io/k3s/pull/7583, then your best bet is to disable the packaged coredns and provide your own manifest.

**Comment by MagicStarTrace** [comment]
How should I operate now to be able to achieve it? 
Thank You
